Alignment/HIPAlignmentAlgorithm/scripts/batch.py

```diff
@@ -0,0 +1,92 @@
+#!/bin/env python
```

Comment:
@usarica please rename this file to have a hippy-specific prefix as the scripts in the `scripts` directory are in the global CMSSW path, i.e. if this goes into the release, every CMSSW user will have an executable `batch.py`.
Renaming it to , e.g., `hippy_batch.py` or `hp_batch.py`would probably avoid confusion for others.